include/openssl/dsa.h

```diff
@@ -85,35 +92,41 @@ DECLARE_ASN1_ENCODE_FUNCTIONS_only(DSA_SIG, DSA_SIG)
 void DSA_SIG_get0(const DSA_SIG *sig, const BIGNUM **pr, const BIGNUM **ps);
 int DSA_SIG_set0(DSA_SIG *sig, BIGNUM *r, BIGNUM *s);
```

Comment:
Hmmm, should the `DSA_SIG` functions remain non-deprecated?  I'd say it's at least up for debate...

Reply:
I'm fine to go either way here.